src/bridge.ts

```diff
@@ -1615,6 +1589,10 @@ export class Bridge {
         return Date.now() - sentTs;
     }
 
+    public updateRoomLinkValidatorRules(rules: Rules): void {
```

Comment:
Is this meant to be manually called by the implementation? I wonder if we could sneak this in to the SIGHUP handler in cli.ts, but I don't quite see us having access to the Bridge itself from there...

Reply:
Yeah, that's the problem :(

The Cli and Bridge are logically separate and it's up to the bridge dev to hook one into the other. It's a bit naff, and I'm not sure it's obvious that you need to do this.

Reply:
The other problem is that since config files are a freeform schema, we wouldn't know which bit to hook into. I've given some information in the changelog, but we can do better.

Reply:
Right. I don't have any better idea, and I've been holding this up for too long :) Maybe some clever idea will come up eventually.